packages/bouncer/build

```diff
@@ -26,3 +26,9 @@ SERVICE_FILE_NAME="dcos-bouncer.service"
 SERVICE_FILE_PATH="$PKG_PATH/dcos.target.wants_master/${SERVICE_FILE_NAME}"
 mkdir -p "$(dirname "$SERVICE_FILE_PATH")"
 envsubst '$PKG_PATH' < "/pkg/extra/${SERVICE_FILE_NAME}" > "${SERVICE_FILE_PATH}"
+
+# Copy the IAM users migration script to the package bin directory.
+mkdir -p $PKG_PATH/bin
```

Comment:
Nit: mkdir -p "$PKG_PATH/bin" ? (to make shellcheck happy: https://github.com/koalaman/shellcheck/wiki/SC2086)

Reply:
I am going to keep this as it is to be consistent with other build scripts.